linera-chain/src/data_types.rs

```diff
@@ -587,16 +587,62 @@ impl<'a> SignatureAggregator<'a> {
     }
 }
 
+fn is_ordered(values: &[(ValidatorName, Signature)]) -> bool {
+    values.windows(2).all(|pair| pair[0].0 <= pair[1].0)
```

Comment:
Duplicate entries should not be allowed:

```suggestion
    values.windows(2).all(|pair| pair[0].0 < pair[1].0)
```